src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        List<Ticket> listOfTickets = new ArrayList<>();
+        for (Ticket ticket : shoppingCart.getTickets()) {
+            listOfTickets.add(ticket.clone());
+        }
```

Comment:
Cloning is overkill solution. We just need to use another list object but not all tickets inside
```suggestion
        List<Ticket> listOfTickets = new ArrayList<>(shoppingCart.getTickets());
```